src/chat.rs

```diff
@@ -3400,6 +3443,9 @@ pub async fn forward_msgs(context: &Context, msg_ids: &[MsgId], chat_id: ChatId)
     Ok(())
 }
 
+/// Resends given messages with the same Message-ID.
```

Comment:
```suggestion
/// Resends given messages with the same database message id.
```

spelling/case "Message-ID" is usually used to refer to RFC 724 Message-ID

Reply:
k, once can read that in both ways :) i thought it is about the parameters :)